kobo/apps/superuser_stats/tasks.py

```diff
@@ -212,6 +212,13 @@ def flatten_metadata_inplace(metadata):
             if isinstance(v, dict) and 'value' in v:
                 metadata[k] = v['value']
 
+    def get_row_value(row, col):
```

Comment:
What about using annotations typing even if it's a small inner function?

Reply:
Good idea 👍